cts-java/src/test/java/org/ga4gh/cts/api/g2p/GenotypePhenotypeSearchIT.java

```diff
@@ -0,0 +1,264 @@
+import org.ga4gh.methods.SearchGenotypePhenotypeResponse;
+import org.ga4gh.models.Dataset;
+import org.ga4gh.models.ExternalIdentifier;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import java.net.HttpURLConnection;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.StrictAssertions.fail;
+import static org.ga4gh.cts.api.Utils.catchGAWrapperException;
+
+/**
+ * Tests dealing with searching for datasets.
+ *
+ * @author Brian Walsh
+ */
+@Category(DatasetsTests.class)
+public class GenotypePhenotypeSearchIT {
+
+    private static Client client = new Client(URLMAPPING.getInstance());
+
+    /**
+     *  error if no parameters
+     * @throws AvroRemoteException if there's an unanticipated error
+     */
+    @Test
+    public void ensureCatchNoParametersSearch() throws AvroRemoteException {
+        try {
+            SearchGenotypePhenotypeRequest request = SearchGenotypePhenotypeRequest.newBuilder().build();
+            SearchGenotypePhenotypeResponse response = client.genotypePhenotype.searchGenotypePhenotype(request);
+            fail("should have thrown error");
+        } catch ( Exception e) {
+            // no-op, expected
+        }
+    }
+
+    /**
+     * search for feature
+     * @throws AvroRemoteException
+     */
+    @Test
+    public void simpleFeatureSearch() throws AvroRemoteException {
+        SearchGenotypePhenotypeRequest request = SearchGenotypePhenotypeRequest.newBuilder().build();
+        request.setFeature("KIT *wild");
+        SearchGenotypePhenotypeResponse response = client.genotypePhenotype.searchGenotypePhenotype(request);
+        assertThat(response).isNotNull();
+        assertThat(response.getAssociations()).isNotNull() ;
+        assertThat(response.getAssociations().get(0)).isNotNull() ;
+    }
+
+
+    /**
+     * search for evidence
+     * @throws AvroRemoteException
+     */
+    @Test
+    public void simpleEvidenceSearch() throws AvroRemoteException {
+        SearchGenotypePhenotypeRequest request = SearchGenotypePhenotypeRequest.newBuilder().build();
+        request.setEvidence("imatinib");
+        SearchGenotypePhenotypeResponse response = client.genotypePhenotype.searchGenotypePhenotype(request);
+        assertThat(response).isNotNull();
+        assertThat(response.getAssociations()).isNotNull() ;
+        assertThat(response.getAssociations().get(0)).isNotNull() ;
+    }
+
+
+
+    /**
+     * search for Phenotype
+     * @throws AvroRemoteException
+     */
+    @Test
+    public void simplePhenotypeSearch() throws AvroRemoteException {
+        SearchGenotypePhenotypeRequest request = SearchGenotypePhenotypeRequest.newBuilder().build();
+        request.setPhenotype("GIST");
+        SearchGenotypePhenotypeResponse response = client.genotypePhenotype.searchGenotypePhenotype(request);
+        assertThat(response).isNotNull();
```

Comment:
As above, this should not be `null`.  If it can be null, the `Client.searchGenotypePhenotype` method, or the server endpoint, are behaving in a manner unlike the other `Client`` methods or server endpoints.